vaadinfrontend/frontend/themes/datamanager/components/card.css

```
@@ -227,3 +226,74 @@
   list-style-type: none;
   display: contents;
 }
+
+/* Support section formatting and components */
+
+.experiment-item-collection {
+  display: grid;
+  row-gap: 1rem;
+  grid-auto-rows: max-content;
+}
```

NIT: We can also be flexible here: 
```suggestion
.experiment-item-collection {
 display: flex;
flex-direction: column;
}

---

but then we loose the row-gap i explicitly defined to fit the UI design and give it more space :)

---

Oh you can also add the row-gap parameter this was a copy paste error on my part